feature/session/src/main/res/layout/item_session.xml

```diff
@@ -80,6 +80,7 @@
             android:singleLine="false"
             android:textAppearance="?attr/textAppearanceHeadline6"
             android:textColor="@color/session_title"
+            android:lineSpacingExtra="8sp"
```

Comment:
I guess session title need more spacing on figma. Maybe it should be `16sp` or `12sp` ?

Reply:
@koogawa Could you tweak it as a new Pull Request?

Reply:
That's a good point!